### PR TITLE
Visa sponsorship salary thresholds updated

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
Visa sponsorship salary thresholds updated

### Trello card

https://trello.com/c/lnKAEH1g/5853-visa-sponsorship-salary-threshold-update

### CODE REVIEW

File: app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md [hunk 1]
```diff
@@ -99,25 +99,25 @@ The salary you must be paid to get a visa varies according to  whether you are e
...
 If you are employed as an unqualified teacher and work in:
 
-* inner London, you must earn £24,254
-* outer London, you must earn £22,924
-* London fringe, you must earn £20,960
-* the rest of England, you must earn £20,960
+* inner London, you must earn £25,831
+* outer London, you must earn £24,415
+* London fringe, you must earn £23,200
+* the rest of England, you must earn £23,200
```
Comment (MaxineCl) [inline app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md:112]: @laurahermionetennant92 this is the same amount as for London fringe. Is that correct?

File: app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md [hunk 2]
```diff
@@ -130,14 +130,14 @@ You may be able to train part-time, but your salary must meet the threshold for
...
 The full-time equivalent of your salary must meet the threshold set for the skilled worker visa in your region, as follows: 
 
-* inner London: £23,099
-* outer London: £21,832
-* London fringe: £20,960
-* rest of England: £20,960
+* inner London: £25,831
+* outer London: £24,415
+* London fringe: £23,200
+* rest of England: £23,200
```
Comment (MaxineCl) [inline app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md:140]: @laurahermionetennant92 same as above comments re if the amount should be the same for London fringe and rest of England.

File: app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md [hunk 3]
```diff
@@ -99,25 +99,25 @@ The salary you must be paid to get a visa varies according to  whether you are e
 
 If you are employed as a qualified teacher and work in:
 
-* inner London, you must earn £34,502
-* outer London, you must earn £32,407
-* London fringe, you must earn £29,344
-* the rest of England, you must earn £28,000
+* inner London, you must earn £36,745
+* outer London, you must earn £34,514
+* London fringe, you must earn £31,350
```
Comment (MaxineCl) [inline app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md:104]: @laurahermionetennant92 what's the difference between outer London and London fringe?

Comment (laurahermionetennant92) [reply app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md:104]: No idea haha. But schools will know which category they fall into, which is why we suggest teachers check with them.

**Review by MaxineCl — APPROVED** [review]

**Pull request merged**